video/out/gpu/video.c

```diff
@@ -3078,7 +3078,9 @@ static void pass_draw_to_screen(struct gl_video *p, const struct ra_fbo *fbo, in
         if (p->opts.alpha_mode == ALPHA_BLEND_TILES) {
             // Draw checkerboard pattern to indicate transparency
             GLSLF("// transparency checkerboard\n");
-            GLSL(bvec2 tile = lessThan(fract(gl_FragCoord.xy * 1.0/32.0), vec2(0.5));)
+            GLSLF("vec2 tile_coord = vec2(gl_FragCoord.x, %d + %f * gl_FragCoord.y);",
```

Comment:
I think this has to be `%d.0` because GLSL doesn't permit fixing integer and floats

Reply:
It gets auto-converted in every sane version of GLSL. Notably excluding GLES 2. (Though I just dropped GLES 2 support altogether in libplacebo for reasons like this)